app/Resources/views/todo/index.html.twig

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```

Comment:
It's nicer if you make a base twig template, then you can have all your other templates extend it, that way you don't have to duplicate boilerplate HTML and all your pages will look the same.